aspnetcore/blazor/fundamentals/signalr.md

```diff
@@ -650,27 +650,43 @@ By default, components are prerendered on the server before the client connectio
 
 Monitor inbound circuit activity using the <xref:Microsoft.AspNetCore.Components.Server.Circuits.CircuitHandler.CreateInboundActivityHandler%2A> method on <xref:Microsoft.AspNetCore.Components.Server.Circuits.CircuitHandler>. Inbound circuit activity is any activity sent from the browser to the server, such as UI events or JavaScript-to-.NET interop calls.
 
-For example, you can use a circuit activity handler to detect if the client is idle:
+For example, you can use a circuit activity handler to detect if the client is idle and log its circuit ID (<xref:Microsoft.AspNetCore.Components.Server.Circuits.Circuit.Id?displayProperty=nameWithType>):
 
 ```csharp
+using Microsoft.AspNetCore.Components.Server.Circuits;
+using Microsoft.Extensions.Options;
+using Timer = System.Timers.Timer;
```

Comment:
Is `using System.Timers;` insufficient? Or does that create ambiguity in the code somewhere?

Reply:
I took that from Dan's code. It looks like going with `using System.Timers.Timer;` is ambiguous with `System.Threading.Timer`. I suppose that's why he did it this way.

Here's Dan's code btw ---

https://devblogs.microsoft.com/dotnet/asp-net-core-updates-in-dotnet-8-preview-3/#monitor-blazor-server-circuit-activity

Reply:
In that case, it's fine how it is!